### ISSUE TITLE
Shares getting " (2)" appended.

### ISSUE THREAD

**Issue by ianaharding** [owncloud/core#25997]
Apparently this has happened before.

https://github.com/owncloud/core/issues/10736

I updated and this started happening.  Not sure if it's related, but two apps, Calendar and Contacts say they have an update pending, I push the button to update and it returns but still says it needs updated.

### Steps to reproduce
1.  Update
2.  See the problem.
3.

### Expected behaviour
Tell us what should happen

After update it should continue to work.
### Actual behaviour
Tell us what happens instead
I see (2) multiple times after some share names.
### Server configuration
**Operating system**:
CentOS 7 3.10.0-327.28.3.el7.x86_64 #1 SMP Thu Aug 18 19:05:49 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

**Web server:**
httpd-2.4.6-40.el7.centos.4.x86_64

**Database:**
mariadb-5.5.50-1.el7_2.x86_64

**PHP version:**
php-5.4.16-36.3.el7_2.x86_64

**ownCloud version:** (see ownCloud admin page)
Version

ownCloud 9.1.0 (stable)
**Updated from an older ownCloud or fresh install:**

**Where did you install ownCloud from:**

**Signing status (ownCloud 9.0 and above):**

Example Domain

This domain is established to be used for illustrative examples in documents. You may use this domain in examples without prior coordination or asking for permission.

**List of activated apps:**

```
Enabled:
  - activity: 2.3.2
  - calendar: 1.1
  - comments: 0.3.0
  - contacts: 1.2.0.0
  - dav: 0.2.5
  - federatedfilesharing: 0.3.0
  - federation: 0.1.0
  - files: 1.5.1
  - files_pdfviewer: 0.8.1
  - files_sharing: 0.10.0
  - files_texteditor: 2.1
  - files_trashbin: 0.9.0
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - gallery: 15.0.0
  - notifications: 0.3.0
  - provisioning_api: 0.5.0
  - systemtags: 0.3.0
  - templateeditor: 0.1
  - updatenotification: 0.2.1
  - user_ldap: 0.9.0
Disabled:
  - encryption
  - external
  - files_antivirus
  - files_external
  - firstrunwizard
  - user_external
```

**The content of config/config.php:**

```
{
    "system": {
        "instanceid": "ocfg33ro8d1p",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "localhost",
            "share.brownpapertickets.com",
            "share-sea.brownpapertickets.com"
        ],
        "datadirectory": "\/var\/www\/html\/owncloud\/data",
        "overwrite.cli.url": "http:\/\/localhost:8888\/owncloud",
        "dbtype": "mysql",
        "version": "9.1.0.15",
        "dbname": "owncloud",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "logtimezone": "UTC",
        "installed": true,
        "theme": "",
        "loglevel": 0,
        "maintenance": false,
        "trashbin_retention_obligation": "auto, auto, auto, auto, auto, auto, auto, auto, auto, auto, auto, auto, auto, auto, auto, auto, auto, auto, auto, auto",
        "mail_domain": "brownpapertickets.com",
        "mail_from_address": "owncloud",
        "mail_smtpdebug": true,
        "mail_smtpmode": "smtp",
        "mail_smtphost": "127.0.0.1",
        "mail_smtpport": 25,
        "mail_smtptimeout": 10,
        "mail_smtpsecure": "",
        "mail_smtpauth": false,
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "ldapIgnoreNamingRules": false,
        "enforce_home_folder_naming_rule": true,
        "filelocking.enabled": "true",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "localhost",
            "port": 6379,
            "timeout": 0
        },
        "updatechecker": false,
        "htaccess.RewriteBase": "\/"
    },
    "apps": {
        "activity": {
            "enabled": "yes",
            "installed_version": "2.3.2",
            "types": "filesystem"
        },
        "admin_migrate": {
            "enabled": "no",
            "installed_version": "0.1"
        },
        "backgroundjob": {
            "lastjob": "5170"
        },
        "calendar": {
            "enabled": "yes",
            "installed_version": "1.1",
            "ocsid": "168707",
            "signed": "true",
            "types": ""
        },
        "comments": {
            "enabled": "yes",
            "installed_version": "0.3.0",
            "types": "logging"
        },
        "contacts": {
            "backend_ldap": "false",
            "enabled": "yes",
            "installed_version": "1.2.0.0",
            "ocsid": "168708",
            "types": ""
        },
        "core": {
            "backgroundjobs_step": "queued_tasks",
            "backgroundjobs_task": "OC_Cache_FileGlobal-gc",
            "global_cache_gc_lastrun": "1451166709",
            "incoming_server2server_share_enabled": "yes",
            "installedat": "1363831784.861",
            "lastcron": "1472664021",
            "lastupdateResult": "[]",
            "lastupdatedat": "1472584256",
            "oc.integritycheck.checker": "[]",
            "outgoing_server2server_share_enabled": "yes",
            "public_caldav": "calendar\/share.php",
            "public_calendar": "calendar\/share.php",
            "public_files": "files_sharing\/public.php",
            "public_gallery": "gallery\/sharing.php",
            "public_webdav": "dav\/appinfo\/v1\/publicwebdav.php",
            "remote_ampache": "media\/remote.php",
            "remote_caldav": "dav\/appinfo\/v1\/caldav.php",
            "remote_calendar": "dav\/appinfo\/v1\/caldav.php",
            "remote_carddav": "dav\/appinfo\/v1\/carddav.php",
            "remote_contacts": "dav\/appinfo\/v1\/carddav.php",
            "remote_contactthumbnail": "contacts\/thumbnail.php",
            "remote_core.css": "\/core\/minimizer.php",
            "remote_core.js": "\/core\/minimizer.php",
            "remote_dav": "dav\/appinfo\/v2\/remote.php",
            "remote_files": "dav\/appinfo\/v1\/webdav.php",
            "remote_filesync": "files\/appinfo\/filesync.php",
            "remote_webdav": "dav\/appinfo\/v1\/webdav.php",
            "repairlegacystoragesdone": "yes",
            "shareapi_allow_mail_notification": "yes",
            "shareapi_allow_public_notification": "yes",
            "umgmt_send_email": "true",
            "umgmt_show_backend": "true",
            "umgmt_show_email": "true",
            "umgmt_show_last_login": "true",
            "umgmt_show_storage_location": "true"
        },
        "dav": {
            "enabled": "yes",
            "installed_version": "0.2.5",
            "types": "filesystem"
        },
        "federatedfilesharing": {
            "enabled": "yes",
            "installed_version": "0.3.0",
            "types": ""
        },
        "federation": {
            "enabled": "yes",
            "installed_version": "0.1.0",
            "types": "authentication"
        },
        "files": {
            "cronjob_scan_files": "500",
            "enabled": "yes",
            "installed_version": "1.5.1",
            "types": "filesystem"
        },
        "files_imageviewer": {
            "enabled": "no",
            "installed_version": "1.0"
        },
        "files_odfviewer": {
            "enabled": "no",
            "installed_version": "0.1"
        },
        "files_pdfviewer": {
            "enabled": "yes",
            "installed_version": "0.8.1",
            "ocsid": "166049",
            "types": ""
        },
        "files_sharing": {
            "enabled": "yes",
            "incoming_server2server_share_enabled": "yes",
            "installed_version": "0.10.0",
            "outgoing_server2server_share_enabled": "yes",
            "types": "filesystem"
        },
        "files_texteditor": {
            "enabled": "yes",
            "installed_version": "2.1",
            "ocsid": "166051",
            "types": ""
        },
        "files_trashbin": {
            "enabled": "yes",
            "installed_version": "0.9.0",
            "types": "filesystem"
        },
        "files_versions": {
            "enabled": "yes",
            "installed_version": "1.3.0",
            "types": "filesystem"
        },
        "files_videoplayer": {
            "enabled": "yes",
            "installed_version": "0.9.8",
            "types": ""
        },
        "files_videoviewer": {
            "enabled": "no",
            "installed_version": "0.1.1"
        },
        "firstrunwizard": {
            "enabled": "no",
            "installed_version": "1.0"
        },
        "gallery": {
            "enabled": "yes",
            "installed_version": "15.0.0",
            "types": ""
        },
        "media": {
            "enabled": "no",
            "installed_version": "0.4.3"
        },
        "notifications": {
            "enabled": "yes",
            "installed_version": "0.3.0",
            "types": "logging"
        },
        "provisioning_api": {
            "enabled": "yes",
            "installed_version": "0.5.0",
            "types": "prevent_group_restriction"
        },
        "search_lucene": {
            "enabled": "no",
            "installed_version": "0.5.3",
            "ocsid": "168709",
            "types": "filesystem"
        },
        "systemtags": {
            "enabled": "yes",
            "installed_version": "0.3.0",
            "types": "logging"
        },
        "tasks": {
            "enabled": "no",
            "installed_version": "0.1",
            "types": ""
        },
        "templateeditor": {
            "enabled": "yes",
            "installed_version": "0.1",
            "types": ""
        },
        "updatenotification": {
            "calendar": "1.3.3",
            "contacts": "1.3.1.0",
            "enabled": "yes",
            "installed_version": "0.2.1",
            "types": ""
        },
        "updater": {
            "79a093857e575da22e8a5ca5468559fb": "",
            "enabled": "no",
            "fe90f5fe081e0ec25020c47b250328ff": "",
            "installed_version": "0.4",
            "ocsid": "166059",
            "types": ""
        },
        "user_ldap": {
            "cleanUpJobOffset": "100",
            "enabled": "yes",
            "enforce_home_folder_naming_rule": "",
            "has_memberof_filter_support": "1",
            "home_folder_naming_rule": "",
            "installed_version": "0.9.0",
            "last_jpegPhoto_lookup": "0",
            "ldap_agent_password": "WGlPcEJjX0Zfd2pkcTY4",
            "ldap_attributes_for_group_search": "",
            "ldap_attributes_for_user_search": "",
            "ldap_backup_host": "",
            "ldap_backup_port": "",
            "ldap_base": "dc=bpt,dc=rocks",
            "ldap_base_groups": "cn=groups,cn=accounts,dc=bpt,dc=rocks",
            "ldap_base_users": "cn=users,cn=accounts,dc=bpt,dc=rocks",
            "ldap_cache_ttl": "600",
            "ldap_configuration_active": "1",
            "ldap_display_name": "displayName",
            "ldap_dn": "uid=owncloud_daemon,cn=users,cn=accounts,dc=bpt,dc=rocks",
            "ldap_dynamic_group_member_url": "",
            "ldap_email_attr": "mail",
            "ldap_experienced_admin": "0",
            "ldap_expert_username_attr": "",
            "ldap_expert_uuid_group_attr": "cn",
            "ldap_expert_uuid_user_attr": "uid",
            "ldap_group_display_name": "cn",
            "ldap_group_filter": "(&(|(objectclass=ipausergroup)))",
            "ldap_group_filter_mode": "1",
            "ldap_group_member_assoc_attribute": "uniqueMember",
            "ldap_groupfilter_groups": "[object Window]",
            "ldap_groupfilter_objectclass": "ipausergroup",
            "ldap_host": "ldaps:\/\/10.1.1.52",
            "ldap_login_filter": "(&(objectclass=person)(uid=%uid))",
            "ldap_login_filter_mode": "0",
            "ldap_loginfilter_attributes": "",
            "ldap_loginfilter_email": "0",
            "ldap_loginfilter_username": "1",
            "ldap_nested_groups": "0",
            "ldap_override_main_server": "",
            "ldap_paging_size": "500",
            "ldap_port": "636",
            "ldap_quota_attr": "",
            "ldap_quota_def": "",
            "ldap_tls": "0",
            "ldap_turn_off_cert_check": "1",
            "ldap_user_display_name_2": "",
            "ldap_user_filter_mode": "0",
            "ldap_userfilter_groups": "",
            "ldap_userfilter_objectclass": "",
            "ldap_userlist_filter": "(objectclass=person)",
            "types": "authentication",
            "use_memberof_to_detect_membership": "1"
        },
        "user_migrate": {
            "enabled": "no",
            "installed_version": "0.1"
        }
    }
}

```

**Are you using external storage, if yes which one:** local/smb/sftp/...
no
**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
LDAP
#### LDAP configuration (delete this part if not used)

```
+-------------------------------+----------------------------------------------------------+
| Configuration                 |                                                          |
+-------------------------------+----------------------------------------------------------+
| hasMemberOfFilterSupport      | 1                                                        |
| hasPagedResultSupport         |                                                          |
| homeFolderNamingRule          |                                                          |
| lastJpegPhotoLookup           | 0                                                        |
| ldapAgentName                 | uid=owncloud_daemon,cn=users,cn=accounts,dc=,dc= |
| ldapAgentPassword             | ***                                                      |
| ldapAttributesForGroupSearch  |                                                          |
| ldapAttributesForUserSearch   |                                                          |
| ldapBackupHost                |                                                          |
| ldapBackupPort                |                                                          |
| ldapBase                      | dc=bpt,dc=rocks                                          |
| ldapBaseGroups                | cn=groups,cn=accounts,dc=,dc=                    |
| ldapBaseUsers                 | cn=users,cn=accounts,dc=,dc=                     |
| ldapCacheTTL                  | 600                                                      |
| ldapConfigurationActive       | 1                                                        |
| ldapDynamicGroupMemberURL     |                                                          |
| ldapEmailAttribute            | mail                                                     |
| ldapExperiencedAdmin          | 0                                                        |
| ldapExpertUUIDGroupAttr       | cn                                                       |
| ldapExpertUUIDUserAttr        | uid                                                      |
| ldapExpertUsernameAttr        |                                                          |
| ldapGroupDisplayName          | cn                                                       |
| ldapGroupFilter               | (&(|(objectclass=ipausergroup)))                         |
| ldapGroupFilterGroups         | [object Window]                                          |
| ldapGroupFilterMode           | 1                                                        |
| ldapGroupFilterObjectclass    | ipausergroup                                             |
| ldapGroupMemberAssocAttr      | uniqueMember                                             |
| ldapHost                      | ldaps://10.1.1.1                                       |
| ldapIgnoreNamingRules         |                                                          |
| ldapLoginFilter               | (&(objectclass=person)(uid=%uid))                        |
| ldapLoginFilterAttributes     |                                                          |
| ldapLoginFilterEmail          | 0                                                        |
| ldapLoginFilterMode           | 0                                                        |
| ldapLoginFilterUsername       | 1                                                        |
| ldapNestedGroups              | 0                                                        |
| ldapOverrideMainServer        |                                                          |
| ldapPagingSize                | 500                                                      |
| ldapPort                      | 636                                                      |
| ldapQuotaAttribute            |                                                          |
| ldapQuotaDefault              |                                                          |
| ldapTLS                       | 0                                                        |
| ldapUserDisplayName           | displayName                                              |
| ldapUserDisplayName2          |                                                          |
| ldapUserFilter                | (objectclass=person)                                     |
| ldapUserFilterGroups          |                                                          |
| ldapUserFilterMode            | 0                                                        |
| ldapUserFilterObjectclass     |                                                          |
| ldapUuidGroupAttribute        | auto                                                     |
| ldapUuidUserAttribute         | auto                                                     |
| turnOffCertCheck              | 1                                                        |
| useMemberOfToDetectMembership | 1                                                        |
+-------------------------------+----------------------------------------------------------+

```

### Client configuration
**Browser:**
Firefox ESR 43
**Operating system:**
CentOS7
#### Web server error log
```
[Sun Aug 28 05:46:16.969035 2016] [auth_digest:notice] [pid 2854] AH01757: generating secret for digest authentication ...
[Sun Aug 28 05:46:16.970704 2016] [lbmethod_heartbeat:notice] [pid 2854] AH02282: No slotmem from mod_heartmonitor
[Sun Aug 28 05:46:16.972691 2016] [:warn] [pid 2854] NSSSessionCacheTimeout is deprecated. Ignoring.
[Sun Aug 28 05:46:17.029722 2016] [mpm_prefork:notice] [pid 2854] AH00163: Apache/2.4.6 (CentOS) OpenSSL/1.0.1e-fips mod_nss/2.4.6 NSS/3.19.1 Basic ECC PHP/5.4.16 configured -- resuming normal operations
[Sun Aug 28 05:46:17.029753 2016] [core:notice] [pid 2854] AH00094: Command line: '/usr/sbin/httpd -D FOREGROUND'

```

#### ownCloud log (data/owncloud.log)
```
It is immense.  I turned on LDAP debugging so it's spewing stuff.```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by RealRancor** [comment]
@ianaharding Might be a duplicate of https://github.com/owncloud/core/issues/24575 which gets fixed with the upcoming 9.1.1

**Comment by PVince81** [comment]
Please try again with 9.1.1RC1
